### PR TITLE
[zh-cn]Update dns-debugging-resolution.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:
https://github.com/kubernetes/website/pull/35507/files?short_path=6bbe991#diff-6bbe9914dacbad626869c3fbbdd483e1a3ffd24d2c56565aa21e50e4b37043a7

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ busybox   1/1       Running   0          <some-time>
 {{< codenew file="admin/dns/dnsutils.yaml" >}}
 
 {{< note >}}
-此示例在 `default` 命名空间创建 pod。 服务的 DNS 名字解析取决于 pod 的命名空间。 详细信息请查阅
+此示例在 `default` 名字空间创建 pod。
+服务的 DNS 名字解析取决于 pod 的名字空间。 详细信息请查阅
 [服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:59]:
```suggestion
服务的 DNS 名字解析取决于 pod 的名字空间。
详细信息请查阅[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 2]
```diff
@@ -277,10 +296,11 @@ command.
 -->
 ### DNS 的端点公开了吗？    {#are-dns-endpoints-exposed}
 
-你可以使用 `kubectl get endpoints` 命令来验证 DNS 的端点是否公开了。
+你可以使用 `kubectl get endpoints` 命令来验证
+DNS 的端点是否公开了。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:300]:
```suggestion
你可以使用 `kubectl get endpoints` 命令来验证 DNS 的端点是否公开了。
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 3]
```diff
@@ -351,7 +372,8 @@ data:
 <!--
 After saving the changes, it may take up to minute or two for Kubernetes to propagate these changes to the CoreDNS pods.
 -->
-保存这些更改后，你可能会需要等待一到两分钟让 Kubernetes 把这些更改应用到
+保存这些更改后，你可能会需要等待一到两分钟让
+Kubernetes 把这些更改应用到
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:376]:
```suggestion
保存这些更改后，你可能会需要等待一到两分钟让 Kubernetes 把这些更改应用到
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 4]
```diff
@@ -54,8 +54,9 @@ busybox   1/1       Running   0          <some-time>
 {{< codenew file="admin/dns/dnsutils.yaml" >}}
 
 {{< note >}}
-此示例在 `default` 命名空间创建 pod。 服务的 DNS 名字解析取决于 pod 的命名空间。 详细信息请查阅
-[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
+此示例在 `default` 名字空间创建 pod。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:57]:
```suggestion
此示例在 `default` 名字空间创建 Pod。
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 5]
```diff
@@ -54,8 +54,9 @@ busybox   1/1       Running   0          <some-time>
...
+服务的 DNS 名字解析取决于 pod 的名字空间。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:58]:
```suggestion
服务的 DNS 名字解析取决于 Pod 的名字空间。
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 6]
```diff
@@ -54,8 +54,9 @@ busybox   1/1       Running   0          <some-time>
...
+详细信息请查阅[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:59]:
```suggestion
详细信息请查阅 [Pod 与 Service 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)。
```
Sometimes, if the link name is different from the linked title, people may be doubted  whether it opens a wrong page.
https://kubernetes.io/zh-cn/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 7]
```diff
@@ -455,11 +475,11 @@ the namespace of the service.
 
 This query is limited to the pod's namespace:
 -->
-### 你的服务在正确的命名空间中吗？
+### 你的服务在正确的名字空间中吗？
 
-未指定命名空间的 DNS 查询仅作用于 pod 所在的命名空间。
+未指定名字空间的 DNS 查询仅作用于 pod 所在的名字空间。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:480]:
```suggestion
未指定名字空间的 DNS 查询仅作用于 Pod 所在的名字空间。
```

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 8]
```diff
@@ -455,11 +475,11 @@ the namespace of the service.
...
 
-如果 pod 和服务的命名空间不相同，则 DNS 查询必须指定服务所在的命名空间。
+如果 pod 和服务的名字空间不相同，则 DNS 查询必须指定服务所在的名字空间。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:482]:
```suggestion
如果 Pod 和服务的名字空间不相同，则 DNS 查询必须指定服务所在的名字空间。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
If you localize a word such as pod, service, secret, or configmap, it will look like Pod, Service, Secret, or ConfigMap in Chinese.

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 9]
```diff
@@ -455,21 +475,21 @@ the namespace of the service.
...
+### 你的服务在正确的名字空间中吗？
 
-未指定命名空间的 DNS 查询仅作用于 pod 所在的命名空间。
+未指定名字空间的 DNS 查询仅作用于 Pod 所在的名字空间。
 
-如果 pod 和服务的命名空间不相同，则 DNS 查询必须指定服务所在的命名空间。
+如果 Pod 和服务的名字空间不相同，则 DNS 查询必须指定服务所在的名字空间。
 
-该查询仅限于 pod 所在的名称空间：
+该查询仅限于 Pod 所在的名称空间：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:484]: 名称空间->名字空间

File: content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 10]
```diff
@@ -547,16 +567,16 @@ work properly owing to a known issue with Alpine.
...
 
 <!--
-- [Autoscaling the DNS Service in a Cluster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
-- [DNS for Services and Pods](/docs/concepts/services-networking/dns-pod-service/)
+- See [Autoscaling the DNS Service in a Cluster](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
+- Read [DNS for Services and Pods](/docs/concepts/services-networking/dns-pod-service/)
 -->
 - 参阅[自动扩缩集群中的 DNS 服务](/zh-cn/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
-- 阅读[服务和 Pod 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/)
+- 阅读[Pod 与 Service 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md:581]:
```suggestion
- 阅读 [Pod 与 Service 的 DNS](/zh-cn/docs/concepts/services-networking/dns-pod-service/)
```

**Pull request merged**